encoding_header.py

```diff
@@ -0,0 +1,36 @@
+"""
+<Author>
+  Sebastien Awwad /:
+
+<Start Date>
+  March 18, 2016
+
+<Description>
+  This mini module provides the ENCODING_DECLARATION constant, a string that is
+  prepended to code loaded into VirtualNamespace objects. This is prepended to
+  such code in order to specify UTF-8 encoding and prevent certain bugs.
+
+  Because prepending it to code results in distortions in reported line numbers
+  in tracebacks when exceptions occur, this constant is imported in files that
+  would raise such exceptions, so as to subtract the number of lines it
+  contains from reported line numbers.
+
+  Consequently, multiple modules import this, and because they also depend on
+  each other, it was decided to keep this in its own separate module in order
+  to help avoid import loops.
+  
+  As of this writing, it is used by:
+    virtual_namespace.py
+    tracebackrepy.py
+    safe_check.py
+    safe.py
+
+  It is my hope to restructure this sensibly in time.
```

Comment:
When you say restructure, do you mean that this module might be expanded in the future to include more constants?  If so, `encoding_header.py` seems too specific.  If not, it's not clear how much more restructuring is possible here; it holds a single constant for a specific purpose.